### PR TITLE
Fixed small typo in documentation for OpenContainer

### PR DESCRIPTION
**Pull request by mauzepeda**
OpenContainer's openBuilder action callback is used to close the container, and the reverse with closedBuilder.

### CODE REVIEW

File: packages/palette_generator/example/ios/Flutter/flutter_export_environment.sh [hunk 1]
```diff
@@ -1,10 +1,11 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
-export "FLUTTER_ROOT=/Users/gspencer/code/flutter"
-export "FLUTTER_APPLICATION_PATH=/Users/gspencer/code/packages/packages/palette_generator/example"
+export "FLUTTER_ROOT=/Users/mau/Development/Libraries/flutter"
```
Comment (goderbauer) [inline packages/palette_generator/example/ios/Flutter/flutter_export_environment.sh:3]: Can you please revert the changes to this file since they are unrelated to this PR? Thanks.

Comment (mauzepeda) [reply packages/palette_generator/example/ios/Flutter/flutter_export_environment.sh]: Done!

**Comment by goderbauer** [comment]
submit-queue check shows up as red, but the previous PR actually passed post-submit: https://cirrus-ci.com/build/6366528876576768

Submitting this on red to trigger another post-submit run.
